### PR TITLE
chore(style): fit and finish for components and doc site

### LINKED ISSUE #481: Text on a checkbox does not align correctly
**Describe the bug**
Text on a checkbox does not align correctly.  This should match design with full testing across browsers.

**To Reproduce**
Navigate to checkbox page on https://msft-docs.azurewebsites.net/components/checkbox/ 

**Expected behavior**
Should match the expected design.

**Screenshots**
screen shot 2018-05-30 at 7 03 42 am

**Desktop (please complete the following information):**
 - OS: [e.g. **MacOS**, Windows, Linux]
 - Browser [e.g. Edge, Chrome, Safari, **FireFox**]
 - Version [e.g. **Latest**]

**Smartphone (please complete the following information):**
 - Browser [e.g. stock browser, Safari]

**Additional context**

### LINKED ISSUE #483: Button component with anchor element render wider than buttons
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. MacOS, Windows, Linux]
 - Browser [e.g. Edge, Chrome, Safari]

**Smartphone (please complete the following information):**
 - Browser [e.g. stock browser, Safari]

**Additional context**

### PR DESCRIPTION
**Pull request by chrisdholt**
<body>
closes #481, closes #483 ( & other fit and finish)
<footer>
 

For [details on formatting](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance) pull requests.

### CODE REVIEW

File: packages/fast-components-styles-msft/src/checkbox/index.ts [hunk 1]
```diff
@@ -99,12 +97,11 @@ const styles: ComponentStyles<ICheckboxClassNameContract, IDesignSystem> = {
             return config.foregroundColor;
         },
         ...applyTypeRampConfig("t7"),
-        marginLeft: toPx(5),
-        marginTop: toPx(2)
+        marginLeft: toPx(5)
     },
     checkbox_disabled: {
+        cursor: "not-allowed",
         "& $checkbox_input, & $checkbox_label": {
```
Comment (nalogiudice) [inline packages/fast-components-styles-msft/src/checkbox/index.ts]: Add `, & $checkbox_span` to the `"& $checkbox_input, & $checkbox_label"` selector

File: packages/fast-components-styles-msft/src/checkbox/index.ts [hunk 2]
```diff
@@ -99,12 +97,11 @@ const styles: ComponentStyles<ICheckboxClassNameContract, IDesignSystem> = {
             return config.foregroundColor;
         },
         ...applyTypeRampConfig("t7"),
-        marginLeft: toPx(5),
-        marginTop: toPx(2)
+        marginLeft: toPx(5)
     },
     checkbox_disabled: {
+        cursor: "not-allowed",
```
Comment (nalogiudice) [inline packages/fast-components-styles-msft/src/checkbox/index.ts]: Are we sure we want this? This isn't the "normal" treatment of a disabled checkbox

File: packages/fast-components-styles-msft/src/checkbox/index.ts (hunk 1 above)
Comment (chrisdholt) [reply packages/fast-components-styles-msft/src/checkbox/index.ts]: I moved the opacity to the top level to reduce the selector chain.

File: packages/fast-components-styles-msft/src/checkbox/index.ts (hunk 2 above)
Comment (chrisdholt) [reply packages/fast-components-styles-msft/src/checkbox/index.ts]: @nalogiudice Confirmed with @Falkicon that we'll move forward this currently and revisit the overall story later on.

**Pull request merged**